src/devices/bma400.cpp

```diff
@@ -19,7 +19,7 @@
 
 #define READ_WRITE_LENGTH UINT8_C(46)
 struct bma400_dev bma;
-float accelT = 0, accelX = 0, accelY = 0, accelZ = 0;
+float accelT = 0.0f, accelX = 0.0f, accelY = 0.0f, accelZ = 0.0f;
```

Comment:
I assume you did not use a RegEx to discover all those... Which compiler / linter did you use?